### PR TITLE
Collapse wikis submodule into core

### PR DESCRIPTION
**Pull request by PaulMorris**
I used a bash script (see [stackoverflow answer](https://stackoverflow.com/questions/1759587/un-submodule-a-git-submodule/43554035#43554035), [the code](https://github.com/jeremysears/scripts/blob/master/bin/git-submodule-rewrite), and [prior art blog post](http://x3ro.de/2013/09/01/Integrating-a-submodule-into-the-parent-repository.html)) to collapse the wikis submodule into the core repo (now 'librehq' formerly 'librehq-core'), while preserving the git history.  That's most of the commits here.  Then I made several commits (those on December 11) to integrate the wikis service into librehq.  

I'm not quite sure about how database migrations for the wikis service works in this new arrangement.  The README.md might need to be reworked a bit on that topic.  

I tested by running the steps described in the README.md and everything worked as it had before doing this merge.  (However, database migrations had already been run on my databases.  I didn't redo the databases or migrations.)

### CODE REVIEW

**Comment by frankduncan** [comment]
This looks good @PaulMorris , thanks!  Take a quick look at my final commit before merging, please, as it has the new DB stuff.  You can also try it by blowing away your databases and then recreating them using the createdb commands, then running the migrate command.

**Review by frankduncan — APPROVED** [review]
See previous comment :)

File: README.md [hunk 1]
```diff
@@ -76,15 +76,6 @@ Run flask-migrate migrations
...
 
-Then run flask-migrate for each of the services.  We do it this way so each
-service can handle its own database setup.
-
-```ShellSession
-for service in wikis ; do
-  (cd $service ; FLASK_APP=$service pipenv run flask db upgrade)
-done
-```
-
```
Comment (PaulMorris) [inline README.md:95]: This should either go in its own commit, or be added to one of the others.

**Review by PaulMorris — COMMENTED** [review]
Thanks @frankduncan !  I looked over your commits, then deleted my databases and recreated them, running the new migration script, all as described in the README.  Everything went well.  I was able to create a new LibreHQ user and then log in and do things like change my email address, create a (stub of a) new wiki, etc.

I had one optional suggestion on moving a README edit to a different commit.  And I think this is ready to merge into the mvp-dev branch.  (I can't click approve because I opened the PR.)

Oh, here's the log from the migrations step.  (Not sure if you're seeing these warnings too.  They're not important at this stage.)
```
$ FLASK_APP=librehq pipenv run flask db upgrade
/home/paul/.local/share/virtualenvs/librehq-JgtYcA4H/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:794: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
/home/paul/.local/share/virtualenvs/librehq-JgtYcA4H/lib/python3.6/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
INFO  [alembic.env] Migrating database <default>
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 5873a625b0fc, empty message
INFO  [alembic.env] Migrating database wikis
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 5873a625b0fc, empty message
```
